scripts/codeOptimizer/index.js

```diff
@@ -0,0 +1,18 @@
+import getCommentsFromDirectory from "./codeParsers/index.js";
+import generateReport from "./report/generateReport.js";
+import getAbsolutePath from "./helper/getAbsPath.js";
+
+const sourceCodePath = process.argv[2];
```

Comment:
make a config which will have following points (.env.config.js)

1. src path
2. output path
3. files to parse
4. TECH_DEBT_KEYWORDS